.github/workflows/build.yaml

```diff
@@ -22,34 +22,47 @@ concurrency:
 # will skip running `yarn install` if it successfully fetched from cache
 
 jobs:
-  fmt:
+  prettier:
     name: Format with Prettier
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
 
+      - name: Run prettier with actionsx/prettier
+        uses: actionsx/prettier@v2
+        with:
+          args: --check --loglevel=warn .
+
+  doctoc:
```

Comment:
Now that I'm adding this...do we really need to run `doctoc` in CI? Seems like something that would only be run locally, right? Should I remove?

Reply:
I think we want to because it catches when people have not ran it.

Reply:
Or at least that is how we should set it up if we have not already.

Reply:
Ah yup looks like we do so I think this is good.